src/Core/Components/Base/InputHelpers.cs

```diff
@@ -51,17 +50,27 @@ public static string GetMinValue()
         return value;
     }
 
-    internal static void ValidateIntegerInputs(string? max, string? min)
+
+    internal static void ValidateSByteInputs(string max, string min)
```

Comment:
Why arguments are nullable in all other methods and not your new one?

Reply:
`Convert.ToSByte ` not accept null value

Reply:
Docs say yes:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/5bf23592-1992-49f5-b85c-91665f0bc5fc)